### PR TITLE
concurrency: replace legacy Verifies/ASIL tags with comp_req RecordProperty links

### PR DESCRIPTION
**Pull request by Nikhil2206**
## Summary
Part of #522. Replaces the legacy `ASIL`/`Verifies` `RecordProperty` tags in `synchronized_queue_test.cpp` with the current `PartiallyVerifies: comp_req__concurrency__synchronized_queue` linkage convention.

### CODE REVIEW

File: score/concurrency/synchronized_queue_test.cpp [hunk 1]
```diff
@@ -41,10 +41,9 @@ using ::testing::Return;
 
 TEST(SynchronizedQueue, CheckFalseResponseOnPushWhenMaxQueueLengthReached)
 {
-    RecordProperty("ASIL", "QM");
     RecordProperty("TestType", "interface-test");
     RecordProperty("DerivationTechnique", "boundary-values");
-    RecordProperty("Verifies", "::score::platform::aas::lib::concurrency::SynchronizedQueue");
+    RecordProperty("PartiallyVerifies", "comp_req__concurrency__synchronized_queue");
```
[the remark this answers is not in the document]
Comment (Nikhil2206) [reply score/concurrency/synchronized_queue_test.cpp]: fixed

**Review by 4og — APPROVED** [review]

**Review by 4og — APPROVED** [review]

**Review by 4og — APPROVED** [review]
